### PR TITLE
test: migrate spreadsheet ITs from FW8 version

### PR DESCRIPTION
**Pull request by tomivirkki**
Part of https://github.com/vaadin/components-team-tasks/issues/604

Migrate a few ITs from FW8 repo.

### CODE REVIEW

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/CopyPasteCellsIT.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        waitUntil(new ExpectedCondition<Object>() {
+            @Override
+            public Object apply(WebDriver webDriver) {
+                return spreadsheet.getCellAt("E4").getValue()
+                        .equals(expectedValue);
+            }
+        });
+    }
+
+    @Ignore("The new test UI does not have v-debugwindow-tab available")
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/CopyPasteCellsIT.java:47]: I am wondering if we will ever be able to enable this test and whether we should import it or not. 🤔

File: vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/CopyPasteCellsIT.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        for (WebElement message : messages) {
+            if (isUpdateCellValuesJSON(message.getAttribute("innerHTML"))) {
+                String text = message.getAttribute("innerHTML");
+                int startIndex = text.indexOf('(') + 1;
+                int endIndex = text.indexOf(')');
+                return text.substring(startIndex, endIndex);
+            }
+        }
+        return "";
+    }
```
Comment (DiegoCardoso) [inline vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/CopyPasteCellsIT.java:77]: This (now) unused method can be removed.

Comment (tomivirkki) [reply vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/CopyPasteCellsIT.java]: Done

**Review by DiegoCardoso — APPROVED** [review]
👏🏻

**Pull request merged**